gnocchi/tests/base.py

```diff
@@ -175,8 +173,40 @@ def post_account(self, headers, query_string=None, data=None,
         return {}, None
 
 
+class CaptureOutput(fixtures.Fixture):
+    """Optionally capture the output streams.
+
+    .. py:attribute:: stdout
+
+       The ``stream`` attribute from a :class:`StringStream` instance
+       replacing stdout.
+
+    .. py:attribute:: stderr
+
+       The ``stream`` attribute from a :class:`StringStream` instance
+       replacing stderr.
+
+    """
+
+    def setUp(self):
+        super(CaptureOutput, self).setUp()
+        self._stdout_fixture = fixtures.StringStream('stdout')
+        self.stdout = self.useFixture(self._stdout_fixture).stream
+        self.useFixture(fixtures.MonkeyPatch('sys.stdout', self.stdout))
+        self._stderr_fixture = fixtures.StringStream('stderr')
+        self.stderr = self.useFixture(self._stderr_fixture).stream
+        self.useFixture(fixtures.MonkeyPatch('sys.stderr', self.stderr))
+
+
+class BaseTestCase(testcase.TestCase):
+    def setUp(self):
+        super(BaseTestCase, self).setUp()
+        self.useFixture(CaptureOutput())
+        daiquiri.setup(outputs=[])
```

Comment:
I use OS_XXX var to debug, so having an environment variable to disbale CaptureOuput() and setup
daiquiri with stdout or stderr on debug level would be cool.

Like GNOCCHI_DEBUG_TESTS